test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -522,4 +335,49 @@ describe('Deploy and Retrieve', async () => {
   step('Tear down and clean up the testing environment', async () => {
     await testSetup.tearDown();
   });
+
+  const runAndValidateCommand = async (
+    workbench: Workbench,
+    operation: string,
+    fromTo: string,
+    type: string,
+    prefix?: string
+  ): Promise<void> => {
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      `SFDX: ${operation} This Source ${fromTo} Org`,
+      5
+    );
+
+    await validateCommand(workbench, operation, fromTo, type, prefix);
+  };
+  const validateCommand = async (
+    workbench: Workbench,
+    operation: string,
+    fromTo: string,
+    type: string, // Text to identify operation type (if it has source tracking enabled, disabled or if it was a deploy on save)
+    prefix: string = ''
+  ): Promise<void> => {
+    const successNotificationWasFound = await utilities.notificationIsPresentWithTimeout(
+      workbench,
+      `SFDX: ${operation} Source ${fromTo} Org successfully ran`,
+      utilities.TEN_MINUTES
+    );
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify Output tab
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      'Salesforce CLI',
+      `Starting SFDX: ${operation} Source ${fromTo}`,
+      10
+    );
+    utilities.log(
+      `${operation} time ${type} - 3: ` + (await utilities.getOperationTime(outputPanelText!))
+    );
```

Comment:
Is the `- 3` necessary in this `utilities.log()` statement?  It seems like it's being used as a count of number of times the command was run in the E2E test, which loses its meaning if it's always 3.